app/database/models.py

```diff
@@ -405,3 +406,15 @@ def insert_data(target, session: Session, **kw):
 
 
 event.listen(Language.__table__, "after_create", insert_data)
+
+
+class Note(Base):
+    __tablename__ = "notes"
```

Comment:
It's funny - you changed all `"` to `'` but not here :)

Reply:
Thanks - reformat with black